lib/krikri/enricher.rb

```diff
@@ -33,6 +34,13 @@ def self.queue_name
     end
 
     ##
+    # @see Krikri::Activity#generated_entities
+    # @see Krikri::EntityBehavior
+    def entity_behavior
+      Krikri::AggregationEntityBehavior
```

Comment:
Is there a way around hard coding this?  Maybe something like

```ruby
attr_writer :entity_behavior

def entity_behavior
  @entity_behavior ||= Krikri::AggregationEntityBehavior
end
```

This isn't hugely important, since the Enrichments in their current form only work on `ActiveTriples::Resources`; but it would be nice to loosen this dependency.